codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/plugins/DefaultAWSAuthSchemePlugin.kt

```diff
@@ -45,13 +46,21 @@ class DefaultAWSAuthSchemePlugin(
                         "Default${AuthSchemeResolverGenerator.getSdkId(ctx)}AuthSchemeResolver()",
                     )
                     writer.write("config.authSchemes = \$L", AWSAuthUtils(ctx).getModeledAuthSchemesSupportedBySDK(ctx, writer))
-                    writer.write(
-                        "config.awsCredentialIdentityResolver = \$N()",
-                        AWSSDKIdentityTypes.DefaultAWSCredentialIdentityResolverChain,
-                    )
+                    if (ctx.settings.internalClient) {
+                        writer.write(
+                            "config.awsCredentialIdentityResolver = \$N()",
+                            InternalAWSCommonTypes.EmptyAWSCredentialIdentityResolver,
+                        )
+                    } else {
+                        writer.write(
+                            "config.awsCredentialIdentityResolver = \$N()",
+                            AWSSDKIdentityTypes.DefaultAWSCredentialIdentityResolverChain,
+                        )
+                    }
                     if (AuthUtils(ctx).isSupportedAuthScheme(HttpBearerAuthTrait.ID)) {
                         writer.write(
-                            "config.bearerTokenIdentityResolver = try \$N()",
+                            "config.bearerTokenIdentityResolver = try \$N(\$N())",
+                            SmithyIdentityTypes.ClientConfigDefaultBearerTokenIdentityResolver,
                             AWSSDKIdentityTypes.DefaultBearerTokenIdentityResolverChain,
```

Comment:
Above, the DefaultBearerTokenIdentityResolverChain resolver is wrapped in a ClientConfigDefaultBearerTokenIdentityResolver to signify that it was set as a default value.